### PR TITLE
Fix adding IPpresets to database

### PR DESCRIPTION
**Pull request by egedebruin**
The database did not add the IPCameraPresets to the database, this is fixed now.

### CODE REVIEW

**Comment by bvanwijk** [comment]
Looks good, I agree with merging.

File: backend/src/main/java/com/benine/backend/database/MySQLDatabase.java [hunk 1]
```diff
@@ -273,7 +276,7 @@ private void setIpPreset(IPCameraPreset preset, PreparedStatement statement) {
       statement.setInt(9, autoir);
       statement.setInt(10, preset.getId());
     } catch (SQLException e) {
-      logger.log("Presets could not be added.", LogEvent.Type.CRITICAL);
+      logger.log("Presets could not be added IP.", LogEvent.Type.CRITICAL);
```
Comment (NaomidR) [inline backend/src/main/java/com/benine/backend/database/MySQLDatabase.java]: I don't think this is a correct sentence

**Comment by NaomidR** [comment]
I've placed a small comment. After you have changed this I agree with merging.

**Comment by jochem725** [comment]
I agree with merging.

**Pull request merged**